pkg/k8shandler/util.go

```diff
@@ -13,6 +13,15 @@ import (
 	api "github.com/openshift/elasticsearch-operator/pkg/apis/logging/v1"
 )
 
+func getLogLevel(annotations map[string]string) string {
+	if value, found := annotations[loglevelAnnotation]; found {
+		if strings.TrimSpace(value) != "" {
+			return value
```

Comment:
add input verification
log4j supports `DEBUG,INFO,WARN,ERROR,FATAL,TRACE,OFF,ALL`
logrus supports `panic,fatal,error,warn,warning,info,debug,trace`

Reply:
I'm not certain the case matters for either of these libs or that  we need verification here.  I don't see us exiting a deployment because the value was specified wrong.  The consumer of the value hopefully will print a parse error message which is what the proxy does.  Validation really needs to be part of 'admission' but we don't have that